examples/cra-kitchen-sink/src/stories/addon-jest.testresults.json

```diff
@@ -1 +0,0 @@
"column":89},"end":{"line":4,"column":145}},"line":4},"1":{"name":"(anonymous_1)","decl":{"start":{"line":18,"column":18},"end":{"line":18,"column":19}},"loc":{"start":{"line":18,"column":46},"end":{"line":45,"column":34}},"line":18},"2":{"name":"AsyncItemLoader","decl":{"start":{"line":19,"column":11},"end":{"line":19,"column":26}},"loc":{"start":{"line":19,"column":29},"end":{"line":22,"column":3}},"line":19},"3":{"name":"handleLoadItems","decl":{"start":{"line":22,"column":90},"end":{"line":22,"column":105}},"loc":{"start":{"line":24,"column":4},"end":{"line":30,"column":5}},"line":24},"4":{"name":"(anonymous_4)","decl":{"start":{"line":29,"column":17},"end":{"line":29,"column":18}},"loc":{"start":{"line":29,"column":29},"end":{"line":29,"column":112}},"line":29},"5":{"name":"render","decl":{"start":{"line":30,"column":42},"end":{"line":30,"column":48}},"loc":{"start":{"line":32,"column":4},"end":{"line":45,"column":5}},"line":32},"6":{"name":"(anonymous_6)","decl":{"start":{"line":35,"column":48},"end":{"line":35,"column":49}},"loc":{"start":{"line":35,"column":60},"end":{"line":35,"column":94}},"line":35},"7":{"name":"(anonymous_7)","decl":{"start":{"line":44,"column":65},"end":{"line":44,"column":66}},"loc":{"start":{"line":44,"column":78},"end":{"line":44,"column":138}},"line":44},"8":{"name":"(anonymous_8)","decl":{"start":{"line":50,"column":28},"end":{"line":50,"column":29}},"loc":{"start":{"line":50,"column":40},"end":{"line":90,"column":1}},"line":50},"9":{"name":"(anonymous_9)","decl":{"start":{"line":84,"column":58},"end":{"line":84,"column":59}},"loc":{"start":{"line":84,"column":74},"end":{"line":84,"column":140}},"line":84},"10":{"name":"(anonymous_10)","decl":{"start":{"line":91,"column":35},"end":{"line":91,"column":36}},"loc":{"start":{"line":91,"column":47},"end":{"line":94,"column":62}},"line":91},"11":{"name":"(anonymous_11)","decl":{"start":{"line":105,"column":48},"end":{"line":105,"column":49}},"loc":{"start":{"line":105,"column":60},"end":{"line":149,"column":1}},"line":105},"12":{"name":"(anonymous_12)","decl":{"start":{"line":143,"column":58},"end":{"line":143,"column":59}},"loc":{"start":{"line":143,"column":74},"end":{"line":143,"column":140}},"line":143},"13":{"name":"(anonymous_13)","decl":{"start":{"line":150,"column":55},"end":{"line":150,"column":56}},"loc":{"start":{"line":150,"column":67},"end":{"line":153,"column":62}},"line":150}},"branchMap":{"0":{"loc":{"start":{"line":4,"column":97},"end":{"line":4,"column":143}},"type":"cond-expr","locations":[{"start":{"line":4,"column":121},"end":{"line":4,"column":124}},{"start":{"line":4,"column":127},"end":{"line":4,"column":143}}],"line":4},"1":{"loc":{"start":{"line":4,"column":97},"end":{"line":4,"column":118}},"type":"binary-expr","locations":[{"start":{"line":4,"column":97},"end":{"line":4,"column":100}},{"start":{"line":4,"column":104},"end":{"line":4,"column":118}}],"line":4},"2":{"loc":{"start":{"line":19,"column":143},"end":{"line":19,"column":210}},"type"
```

Comment:
Why did you remove it?

Reply:
How my! Will re-add it. no reason to remove it :+1: